libs/components/packages/src/schematics/ng-generate/add-ai-skills/files/skills/migration-resolver/references/angular-debugging.md

```
@@ -0,0 +1,202 @@
+     providers: [
+       // Must provide everything the component injects
+       { provide: SkyModalService, useValue: mockService },
+     ],
+   });
+   ```
+
+## Lifecycle Hook Issues
+
+### Execution Order
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)